test/definitions/ur.csv

```diff
@@ -0,0 +1,18 @@
+0$0 ملی سیکنڈ
+1$1 ملی سیکنڈ
```

Comment:
[Google Translate](https://translate.google.com/#ur/en/1%20%D9%85%D9%84%DB%8C%20%D8%B3%DB%8C%DA%A9%D9%86%DA%88) says this means "1 millimeter", not "1 millisecond". Is this correct?